### ISSUE TITLE
Butterworth filter

### ISSUE THREAD

**Issue by merlinthemagician** [flika-org/flika#51]
Good afternoon,
When attempting to apply a Butterworth filter to a stack of images, the command seems to get stuck, for the example I ran at "Creating process 14/16". This is not necessarily related to the filter itself, it is accompanied by an error message "Too many open windows". Hitting the "Stop" button leads to IndexError exceptions ("list index out of range") and flika needs to be terminated.

Thanks a lot and best wishes
Ivo

P.S.: Traceback at "Creating process 14/16":

Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/flika-0.2.33-py3.9.egg/flika/process/progress_bar.py", line 28, in run
  File "/usr/local/Cellar/python@3.9/3.9.4/Frameworks/Python.framework/Versions/3.9/lib/python3.9/multiprocessing/context.py", line 103, in Queue
  File "/usr/local/Cellar/python@3.9/3.9.4/Frameworks/Python.framework/Versions/3.9/lib/python3.9/multiprocessing/queues.py", line 48, in __init__
  File "/usr/local/Cellar/python@3.9/3.9.4/Frameworks/Python.framework/Versions/3.9/lib/python3.9/multiprocessing/context.py", line 68, in Lock
  File "/usr/local/Cellar/python@3.9/3.9.4/Frameworks/Python.framework/Versions/3.9/lib/python3.9/multiprocessing/synchronize.py", line 162, in __init__
  File "/usr/local/Cellar/python@3.9/3.9.4/Frameworks/Python.framework/Versions/3.9/lib/python3.9/multiprocessing/synchronize.py", line 57, in __init__
OSError: [Errno 24] Too many open files

**Comment by kyleellefsen** [comment]
Hi Ivo,

I just upgraded my python to version 3.9.5 on MacOS Big Sur, created a fresh virtual environment, install all flika dependencies, and am unable to recreate this error. Which deprecated methods are being called?

**Comment by merlinthemagician** [comment]
Hi Kylle,
thanks for looking into this issue. I've tested calling the Butterworth filter again both with a virtual environment as well as my system installation of python 3.9.5 and I wasn't able to reproduce this issue either. I assume that it was related to the sequence of commands that I ran earlier. I believed that this was related to calls to deprecated methods but the Traceback suggests that it is instead related to parallelisation. My apologies, it seems that it is not related to #50.
Thanks a lot and best wishes
Ivo
